src/services/picture-uploader.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { PicturesStore } from 'api/collections';
+import { Picture } from 'api/models';
+import { UploadFS } from 'meteor/jalik:ufs';
+import { _ } from 'meteor/underscore';
+
+@Injectable()
+export class PictureUploader {
+  select(): Promise<File> {
+    return new Promise((resolve, reject) => {
+      UploadFS.selectFile((file: File) => {
+        resolve(file);
```

Comment:
This is how the implementation for `UploadFS.selectFile` looks like:

```js
var input = document.createElement('input');
input.type = 'file';
input.multiple = false;
input.onchange = function (ev) {
  var files = ev.target.files;
  callback.call(UploadFS, files[0]);
};
// Fix for iOS
input.style = 'display:none';
document.body.appendChild(input);
input.click();
```

As you can see, there is no case in a rejection handler whatsoever. This is a fictive promise actually. Just in case, I will wrap it with `try` and `catch` clause.